giovaninppc/AccessibilityHandbook
# [WIP] Add Dark mode documentation

### Summary
- We are adding a new guide to supporting **Dark Mode**

#### Pages

- What is Dark Mode?
- Appearance changes
- Dynamic colors
- Dynamic assets
- Tintable images
- New game 'An alternative'

### How to test it?
- Go to: Colors Guide > Dark mode section
- Check the pages
- Switch between light and dark mode

### Evidences

| Page | Light | Dark |
|--|--|--|
| What is dark mode?   | <img width="506" alt="Screen Shot 2022-10-07 at 18 00 26" src="https://user-images.githubusercontent.com/12978176/194651715-a9639c69-2741-441c-a4a1-8cc8d919bde3.png">    |  <img width="550" alt="Screen Shot 2022-10-07 at 18 00 12" src="https://user-images.githubusercontent.com/12978176/194651729-220db561-1fa0-4ee3-b3f6-4e4019cf7d8b.png">  |
|  Appearence changes  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 00 57" src="https://user-images.githubusercontent.com/12978176/194651833-657483e8-4441-4dc0-9f8f-a7d5934f8fb6.png">   |  <img width="550" alt="Screen Shot 2022-10-07 at 18 01 33" src="https://user-images.githubusercontent.com/12978176/194651865-884a7a8b-1d07-41e2-9d4e-f12aa14d3a5e.png">  |
|  Dynamic colors  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 02 22" src="https://user-images.githubusercontent.com/12978176/194651957-6455e831-408c-48ce-9424-d402a312ffdc.png">  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 02 16" src="https://user-images.githubusercontent.com/12978176/194651942-c0d88e8b-5026-46b4-9254-0530e7f105ab.png">  |
|  Dynamic Assets  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 02 43" src="https://user-images.githubusercontent.com/12978176/194651992-8311a90b-afa9-450b-8e2f-fe16ce925676.png">  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 02 49" src="https://user-images.githubusercontent.com/12978176/194652000-0591a15b-a83f-4d55-8d6d-d6071f7088b3.png">  |
|  Tintable images  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 03 13" src="https://user-images.githubusercontent.com/12978176/194652055-fa563168-9791-49a2-be2e-b1ba25c60154.png">  |  <img width="550" alt="Screen Shot 2022-10-07 at 18 03 07" src="https://user-images.githubusercontent.com/12978176/194652044-7a717b5c-cf38-4d65-bb49-1f4e62bb1b11.png">  |

### Puzzle

On this puzzle, you need to change between light and dark modes in order to enable the switches in the correct order.

|  <img width="550" alt="Screen Shot 2022-10-10 at 10 25 04" src="https://user-images.githubusercontent.com/12978176/194876810-6931934d-c989-4558-ab3f-14dba9e7b484.png">   |    <img width="550" alt="Screen Shot 2022-10-10 at 10 25 08" src="https://user-images.githubusercontent.com/12978176/194876824-140f7db2-aed7-4823-8d5d-80bc5b319712.png">  |
|--|--|

---



closes #21
